apps/docs/.vitepress/plugins/snippetPlugin.ts

```diff
@@ -119,7 +119,12 @@ export const snippetPlugin = (md: MarkdownIt, srcDir: string) => {
           .join('\n')
       );
 
-      const [, partialPath] = filepath.split(/(packages\/.*)/);
+      const match = filepath.match(/(packages|apps)\/(.*)/);
+      const partialPath = match ? match[0] : '';
```

Comment:
Maybe this would suffice.

```suggestion
      const partialPath = match?.[0];
```